atomic_reactor/util.py

```diff
@@ -953,3 +953,25 @@ class LabelFormatter(string.Formatter):
     """
     def get_field(self, field_name, args, kwargs):
         return (self.get_value(field_name, args, kwargs), field_name)
+
+
+def do_retries(method, url, client_statuses=HTTP_CLIENT_STATUS_RETRY,
```

Comment:
Why not use the built-in support for retries provided by `requests`?
https://github.com/projectatomic/osbs-client/blob/master/osbs/http.py#L99-L107